plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -81,6 +81,7 @@ export type StepperProps = {
   components?: {
     ReviewStepComponent?: ComponentType<ReviewStepProps>;
     ReviewStateComponent?: (props: ReviewStateProps) => JSX.Element;
+    backButtonText?: ReactNode;
```

Comment:
I'm wondering if we should deprecate this way eventually, but can probably leave this for now.